src/charm.py

```diff
@@ -96,8 +95,15 @@ def _get_runner_manager(
             token = self.config["token"]
         if path is None:
             path = self.config["path"]
+
         if not token or not path:
             return None
+
+        if "/" in path:
```

Comment:
Should we check whether there are multiple "/" and raise/ return an error if that is the case? Is more than 1 slash an error?